integration-tests/kafka/src/test/java/io/quarkus/it/kafka/streams/KafkaStreamsTest.java

```diff
@@ -114,13 +118,50 @@ public void testKafkaStreams() throws Exception {
         assertCategoryCount(1, 3);
         assertCategoryCount(2, 1);
 
+        testKafkaStreamsAliveAndReady();
+
         // explicitly stopping the pipeline *before* the broker is shut down, as it
         // otherwise will time out
         RestAssured.post("/kafkastreams/stop");
     }
 
+    public void testKafkaStreamsNotAliveAndNotReady() throws Exception {
+        RestAssured.get("/health/ready").then()
+                .statusCode(HttpStatus.SC_SERVICE_UNAVAILABLE)
+                .body(CoreMatchers.containsString("\"name\": \"Kafka Streams topic(s) health check\""))
```

Comment:
Please use jsonpath instead of string assertion.

Something like
`.body("checks.name",  CoreMatchers.is("Kafka Streams topic(s) health check"))`